archunit/src/main/java/com/tngtech/archunit/core/domain/JavaClass.java

```diff
@@ -2343,7 +2342,7 @@ public static DescribedPredicate<JavaClass> resideOutsideOfPackage(String packag
         @PublicAPI(usage = ACCESS)
         public static DescribedPredicate<JavaClass> resideOutsideOfPackages(String... packageIdentifiers) {
             return not(JavaClass.Predicates.resideInAnyPackage(packageIdentifiers))
-                    .as("reside outside of packages ['%s']", Joiner.on("', '").join(packageIdentifiers));
+                    .as("reside outside of packages [%s]", joinSingleQuoted(packageIdentifiers));
         }
 
         /**
```

Comment:
Agree, when adjusting it I also thought that the "this" is a little confusing :thinking: And in general the whole sentence with belong to, because everything seems to deal with classes somehow, but the relationship is somewhat fuzzy from the Javadoc... I tried to improve it.